### DISCUSSION TITLE
possible to add Spide.network

### DISCUSSION THREAD

**Discussion by Raffy142** [engageub/InternetIncome#164] [Ideas]
Hi, is possible add the spide.network?
He have linux and cli, i’m new in this network(spide.network)
Thanks

**Comment by engageub** [comment]
Hello,
Users in the community mentioned that they have not received payment, so this was not added.
If you received payment, do let us know.

https://www.reddit.com/r/spidenetwork/

Thank you

**Comment by TradeBW** [comment]
Is spide still dead ?

**Reply by engageub** [reply]
Based on the reviews and responses, it was decided not to add. 
If you receive payment, let us know.
